src/api/registerForEvents.js

```diff
@@ -11,7 +11,6 @@ export default (auth: Auth) =>
         'update_message',
         'subscription',
         'reaction',
-        'presence',
```

Comment:
What we really want here is to avoid fetching the presence data in the `register` call but still subscribe to the events. @timabbott is working on a backend solution for that I believe